### ISSUE TITLE
PGP WDE OpenCL format fails on nvidia

### ISSUE THREAD

**Issue by magnumripper** [magnumripper/JohnTheRipper#2710]
See https://github.com/magnumripper/JohnTheRipper/pull/2701#issuecomment-324954533 (and follow-ups). Seems to be a driver bug. I'll try rewriting the code without using SHA1-CTX (should be faster as well) when I got the time.

**Comment by magnumripper** [comment]
It was our bug. Fixed in f09bb77a

**Comment by magnumripper** [comment]
It still fails on Super (I did drop all caches, ie. `rm -fr /.nv/ComputeCache`)
```
Device 6: GeForce GTX TITAN X
Benchmarking: pgpwde-opencl [SHA1 OpenCL]... (32xOMP) OpenCL UNKNOWN OPENCL ERROR (-9999) error in common-opencl.c:1820 - Error creating command queue
```
Same with dev 7 (old Titan). This is nvidia 361.28. When I fixed the bug I was running a Titan X on latest Ubuntu LTS nvidia driver, whatever that was, and it worked like a champ (all formats do).

Anyway, dmesg seem to indicate we still have some alignment issue on Super.
